proxy/mempool/mempool_schedule.py

```diff
@@ -285,11 +296,42 @@ def _add_tx_to_sender_pool(self, sender_pool: MPSenderTxPool, tx: MPTxRequest) -
         if sender_pool.len_tx_nonce_queue == 1:
             self._sender_pool_dict[sender_pool.sender_address] = sender_pool
 
+        self._sender_pool_heartbeat_queue.try_add(sender_pool)
```

Comment:
And `try_add` is surplus, because firstly you should remove the `sender-pool` from the queue for the queue reordering.
Check my comment about updating the heartbeat in the MPSenderTxPool::add_tx

Reply:
Yep. Now that the sender pool is being removed, no need to `try_` here.